src/lib.rs

```diff
@@ -63,3 +63,6 @@ pub mod alloc;
 
 #[cfg(feature = "logger")]
 pub mod logger;
+
+#[cfg(all(feature = "exts", feature = "alloc"))]
```

Comment:
Maybe it would be easier for downstream users if we'd introduce a new optional crate feature `fs`, which would automatically activate these features (and any others we'd possibly need in the future)?

Reply:
yes, sounds good!